homeassistant/components/fritz/services.py

```diff
@@ -55,7 +55,7 @@ async def async_call_fritz_service(service_call: ServiceCall) -> None:
         hass.services.async_register(DOMAIN, service, async_call_fritz_service)
 
 
-async def _async_get_configured_avm_device(
+async def _async_get_configured_avm_wrapper(
```

Comment:
```suggestion
async def _async_get_configured_avm_device(
```

Same comment as above